solutions/swb-app/src/schemas/projects/deleteProjectSchema.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+// Schema for deleteAccount API
+import { Schema } from 'jsonschema';
```

Comment:
Should we start using `Zod` instead of `jsonschema`?

Reply:
This file can probably be deleted?

Reply:
This file probably can be deleted. I started this before we had fully converted to `zod` I think.